test/browser/test/viewport.test.ts

```diff
@@ -0,0 +1,26 @@
+import { page, server } from '@vitest/browser/context'
+import { describe, expect, it } from 'vitest'
+
+describe.skipIf(server.provider === 'preview')('viewport window has been properly initialized', () => {
+  it.skipIf(!page.config.browser.headless)('viewport has proper width and size', () => {
```

Comment:
change this test: `'viewport has proper size' or `'viewport has proper width and height'